src/core/announcements/errors.ts

```diff
@@ -21,3 +21,17 @@ export class InvalidAnnouncementTypeError extends AnnouncementError {
     this.announcementType = announcementType;
   }
 }
+
+/**
+ * InvalidHexadecimalSerialization indicates an attempt to coerce a string to hexadecimal serialization failed.
+ */
+export class InvalidHexadecimalSerialization extends DSNPError {
```

Comment:
we might want to move this to `sdk/core/errors` since it's used in more than just the announcement module